src/Process/Runner/InitialTestsRunner.php

```diff
@@ -47,9 +47,14 @@
  */
 final class InitialTestsRunner
 {
+    private const ERROR_TIMEOUT = 10;
+
     private $processBuilder;
     private $eventDispatcher;
 
+    /**
+     * InitialTestsRunner constructor.
+     */
```

Comment:
```suggestion
```

I don't think this comment has any value

Reply:
Removed